Cargo.toml

```diff
@@ -355,10 +355,29 @@ name = "custom-bool"
 path = "examples/derive_ref/custom-bool.rs"
 required-features = ["derive"]
 
+[[example]]
+name = "interop_augment_args"
+path = "examples/derive_ref/1_augment_args.rs"
+required-features = ["derive"]
+
+[[example]]
+name = "interop_augment_subcommands"
+path = "examples/derive_ref/2_augment_subcommands.rs"
+required-features = ["derive"]
+
+[[example]]
+name = "interop_hand_subcommand"
+path = "examples/derive_ref/3_hand_subcommand.rs"
+required-features = ["derive"]
+
+[[example]]
+name = "interop_flatten_hand_args"
+path = "examples/derive_ref/4_flatten_hand_args.rs"
```

Comment:
If we have a leading number, it should be the sections within the document (so if this is inserted as "section 5", then these would be `5_1_...`, `5_2_...`, etc)

Reply:
I felt like they had a sequence in terms of complexity but it may be better to just remove it. I'll do that.

Reply:
BTW, for the file names, should I use kebab-case or snake_case?

Reply:
> BTW, for the file names, should I use kebab-case or snake_case?

While we have one or two kebab-case, looks like we mainly use snake_case.  We should make that more consistent.

Reply:
Ok